lib/MusicBrainz/Server/Data/Role/Collection.pm

```diff
@@ -29,6 +32,69 @@ sub find_by_collection {
     $self->query_to_list_limited($query, [$collection_id], $limit, $offset);
 }
 
+sub find_items_by_collection {
```

Comment:
Wouldn't it make sense to just return the same order in the web service?

Since this duplicates `find_by_collection` other than the ordering and entity creation, it'd seem better if we can just do that, and fix the one spot in the web service to handle `CollectionItems`.

Reply:
The web service should indeed use the same function in the end, I duplicated it on purpose so as to focus on the website first.